tests/layer_tests/common/utils/common_utils.py

```diff
@@ -99,6 +99,11 @@ def allclose(cur_array, ref_array, atol, rtol):
     :return: bool value means that values of tensors are equal with tolerance or not
     """
     if cur_array.dtype.type == str or cur_array.dtype.type == np.str_:
+        # TF can represent string tensors in different format: array of bytestreams
+        # so we have to align formats of both string tensors, for example, to unicode
+        if cur_array.dtype.type != ref_array.dtype.type:
+            cur_array = cur_array.astype('U')
+            ref_array = ref_array.astype('U')
```

Comment:
I've tested this change locally on the Reshape enablement branch, and it fixes most of the test result comparator issues, 12/16 LookupTableFind string tests passed, but 
4 of them failed with error message like:
<img width="323" alt="image" src="https://github.com/openvinotoolkit/openvino/assets/20084299/d71538b1-9257-4309-8ff7-2aa68c313531">

Reply:
Let me create separate PR to enable test cases for `LookupTableFindV2` and I will add required logic for comparison with `bytes` object